packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -1829,6 +1830,53 @@ abstract class MultiSelectableSelectionContainerDelegate extends SelectionContai
     selectable.removeListener(_handleSelectableGeometryChange);
   }
 
+  @override
```

Comment:
should probably also override `getSelectedContent()` so it has the most up to date selection.